db/migrate/20190322140910_create_carts.rb

```diff
@@ -0,0 +1,9 @@
+class CreateCarts < ActiveRecord::Migration[5.2]
+  def change
+    create_table :carts do |t|
```

Comment:
When specifying references to another table, it is often wise to specifically identify foreign keys.  This is something I don't understand very well, so you may want to discuss it with Leila or Ramiro.  The intention is to preserve referential integrity.

Reply:
Leila has requested changes below that mention this. I'll work on it from there.